### PR TITLE
prepare what's new for 4.0

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: docs/source/whatsnew/version4.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- IPython.config (now traitlets.config)
+
+Shims for the deprecated subpackages have been added,
+so existing code should continue to work with a warning.
+
+There are few changes to the code beyond the reorganization and some bugfixes.
+
+IPython highlights:
+
+- Public APIs for discovering IPython paths is moved from :mod:`IPython.utils.path` to :mod:`IPython.paths`. The old function locations
```
Comment (takluyver) [inline docs/source/whatsnew/version4.rst]: end of this sentence missing

File: docs/source/whatsnew/version4.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+===========
+
+Released August, 2015
+
+IPython 4.0 is the first major release after the Big Split.
+IPython no longer contains the notebook, qtconsole, etc. which have moved to
+`jupyter <https://jupyter.readthedocs.org>`_.
+IPython subprojects, such as `IPython.parallel <https://ipyparallel.readthedocs.org>`_ and `widgets <https://ipywidgets.readthedocs.org>`_ have moved to their own repos as well.
+
+The following subpackages are deprecated:
```
Comment (takluyver) [inline docs/source/whatsnew/version4.rst]: Also `IPython.qt`, `IPython.consoleapp`

**Comment by takluyver** [comment]
Minor suggestions, but generally looking good.

**Comment by minrk** [comment]
updated

**Pull request merged**